app/util/tmdb_helpers.py

```diff
@@ -40,4 +46,7 @@ def get_movie(movie_id):
         raise APIException(response.status_code,
                            response.json()['status_message'])
 
-    return {'externalMovieId': movie_id, 'title': response.json().get('title', '')}
\ No newline at end of file
+    movie_data = { desired_key: response.json()[desired_key] for desired_key in DESIRED_KEYS }
```

Comment:
Minor, we could do `.get(desired_key, None)` so that if the response is missing it (for whatever reason) we can fail gracefully

Reply:
I feel like it's better to fail fast rather than wonder why the thing we're expecting isn't there